src/EnergyPlus/WindowManager.cc

```diff
@@ -6915,7 +7141,16 @@ namespace WindowManager {
             }
         }
 
-        NominalConductance = 1.0 / (rOut + Rbare + rIn); // Autodesk:Uninit Rbare was uninitialized if ngllayer > 4
+        NominalConductance = 1.0 / (rOut + Rbare + rIn);
+        Real64 inputU = state.dataMaterial->Material(state.dataConstruction->Construct(ConstrNum).LayerPoint(1)).SimpleWindowUfactor;
+        // only compute adjustment ratio when there is valid user input U
+        if ((WinterSummerFlag == 1)) {
+            if (inputU > 0) {
+                state.dataHeatBal->CoeffAdjRatio(ConstrNum) = inputU / NominalConductance;
+                NominalConductance =
+                    1.0 / (rOut * 1 / state.dataHeatBal->CoeffAdjRatio(ConstrNum) + Rbare + rIn * 1 / state.dataHeatBal->CoeffAdjRatio(ConstrNum));
+            }
+        }
```

Comment:
`NominalU` should be `inputU` in the final report? The adjustment to  `Rbare` is small and ignored.

Reply:
Modified it and applied the adjustment ratio to all terms